src/include/connection.h

```diff
@@ -555,6 +555,13 @@ struct __wt_connection_impl {
     /* Verbose settings for our various categories. */
     WT_VERBOSE_LEVEL verbose[WT_VERB_NUM_CATEGORIES];
 
+/* AUTOMATIC FLAG VALUE GENERATION START 0 */
+#define WT_JSON_OUTPUT_ERROR 0x1u
+#define WT_JSON_OUTPUT_MESSAGE 0x2u
+    /* AUTOMATIC FLAG VALUE GENERATION STOP 8 */
+
```

Comment:
Very minor nit, please remove the blank line. The flags.py script looks looks to validate the STOP value with the definition on the next line (this was added last week).

Reply:
Thanks for raising this! I've push a fix in `5bf2464`